cmd/svcat/plan/get_cmd.go

```diff
@@ -90,23 +100,46 @@ func (c *getCmd) getAll() error {
 		return fmt.Errorf("unable to list classes (%s)", err)
 	}
 
+	if c.classFilter != "" {
+		if !c.lookupByUUID {
+			// Map the external class name to the class name.
+			for _, class := range classes {
+				if c.classFilter == class.Spec.ExternalName {
+					c.classFilter = class.Name
+					break
+				}
+			}
+		}
+		plansFiltered := make([]v1beta1.ClusterServicePlan, 0)
+		for _, p := range plans {
+			if p.Spec.ClusterServiceClassRef.Name == c.classFilter {
+				plansFiltered = append(plansFiltered, p)
+			}
+		}
+		plans = plansFiltered
+	}
+
 	output.WritePlanList(c.Output, plans, classes)
 	return nil
 }
 
 func (c *getCmd) get() error {
 	var plan *v1beta1.ClusterServicePlan
 	var err error
-	if c.lookupByUUID {
+	switch {
+	case c.lookupByUUID:
 		plan, err = c.App.RetrievePlanByID(c.uuid)
-	} else if strings.Contains(c.name, "/") {
+
+	case strings.Contains(c.name, "/"):
```

Comment:
I would prefer to see this string splitting occur in `Validate` as well, so that the get implementation can simply use:

```golang
plan, err = c.App.RetrievePlanByClassAndPlanNames(c.className, c.name)
```